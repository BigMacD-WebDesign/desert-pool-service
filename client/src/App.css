.App {
  text-align: center;
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 80px;
}

.App-header {
  background-color: #222;
  height: 150px;
  padding: 20px;
  color: white;
}

.App-intro {
  font-size: large;
}

.container {
  background: url(assets/images/gunite-spa-inground-pool.jpg) no-repeat center center fixed;
  background-color: rgb(223, 193, 193);
  height: 120vh;
  background-size: cover;
  background-repeat: no-repeat;

}