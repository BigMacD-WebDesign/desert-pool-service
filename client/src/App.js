
// components js files imported
import React, { Component } from "react";
import NavBar from "./components/navbar/NavBar";
import LandingPage from "./components/landingpage/LandingPage";
import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
// import Footer from "./components/footer/Footer"
import Customer from "./components/pages/Customer";
import Login from "./components/pages/Login";
import SignUp from "./components/pages/SignUp";
import ViewCustomer from "./components/pages/ViewCustomer";
import 'bootstrap/dist/css/bootstrap.min.css';
import './App.css';

class App extends Component {

  // Setting this.state.customers to the cutomers db
  // state = {
  //   customers
  // };

  render() {
    return (
  
        <Router basename={process.env.PUBLIC_URL}>
         
          <NavBar />
          <Switch>
          <Route exact path="/" component={LandingPage} />
            <Route exact path="/login" component={Login} />
            <Route exact path="/signup" component={SignUp} />
            <Route exact path="/customer" component={Customer} />
            {/* <Route exact path="/customer" render={(props) => <Customer {...props} customers={this.state.customers} />} /> */}
            <Route exact path="/viewcustomer" component={ViewCustomer} />
            {/* <Route exact path="/contact" component={Contact} /> */}
          </Switch>
          {/* <LandingPage /> */}
          {/* <Footer /> */}
        </Router>
       
     
    );
  };
};
export default App;