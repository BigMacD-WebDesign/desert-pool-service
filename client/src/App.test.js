import React from "react";
import ReactDOM from "react-dom";
import {render, cleanup} from '@testing-library/react'
import App from "./App";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<App />, div);
});

afterEach(cleanup)
 
it('should take a snapshot', () => {
   const { asFragment } = render(<App />)
   
   expect(asFragment(<App />)).toMatchSnapshot()
  })
