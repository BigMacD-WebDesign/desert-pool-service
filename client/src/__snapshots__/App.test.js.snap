// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <nav
    class="navbar navbar navbar-expand-lg navbar-light"
    responisve="true"
  >
    <a
      aria-current="page"
      class="active"
      href="/"
    >
      <span
        class="navbar-brand"
      >
        <span
          class="navbar-title"
        >
          <img
            alt="Desert Pool Service"
            class="d-inline-block align-top"
            height="75px"
            src="Swimming-pool-service-aqua-logo-580x348.jpg"
            width="75px"
          />
        </span>
      </span>
    </a>
    <a
      class="navBarTitle navbar-brand"
      href="/"
    >
      Desert Pool Service
    </a>
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      class="navbar-toggler collapsed"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="navbar-collapse collapse"
      id="basic-navbar-nav"
    >
      <div
        class="navbar-nav"
      />
    </div>
    <button
      class="btn-one btn btn-primary"
      data-testid="login-button"
      type="submit"
    >
      Log In
    </button>
  </nav>
  <div
    class="row section1"
    data-test="row"
  >
    <div
      class="col-md-2"
      data-test="col"
    />
    <div
      class="col-md-8"
      data-test="col"
    >
      <p>
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
        <br />
      </p>
      <p
        class="poolservice"
      />
      <h1>
        Need weekly or seasonal service? Get a FREE Estimate Now!
      </h1>
      <p />
      <a
        class="learn-more"
        href="#moreInfo"
      >
        <button
          class="moreinfo btn btn-primary"
          type="submit"
        >
          More Info
        </button>
      </a>
      <p />
    </div>
  </div>
  <div
    class="row"
    data-test="row"
  >
    <div
      class="col-md-6 text-center formatPic"
      data-test="col"
      fluid="true"
    >
      <p
        id="moreInfo"
      />
      <div
        data-aos="zoom-in-left"
        data-aos-duration="1600"
        data-aos-easing="ease-in-sine"
        data-aos-offset="200"
      >
        <img
          alt="Gazebo Pool"
          height="100%"
          src="a.jpg"
          width="100%"
        />
      </div>
    </div>
    <div
      class="col-md-6 imageBKGR text-center"
      data-test="col"
      fluid="true"
      id="#infobutton"
    >
      <br />
      <h2>
        Your Local pool professional
      </h2>
      <h4>
        We enjoy what we do so you can enjoy your pool!
      </h4>
      <br />
      <h6
        class="H6FontFormat"
      >
        Desert Pool Service is the premier Gilbert pool repair service company. We maintain reasonable prices and spend the necessary time making sure we get the job done the first time around. If you've been delaying much-needed repairs or service for your swimming pool, look no further than Desert Pool Service. As an established service and repair company in Gilbert, our goal is to provide you with quality products and services at affordable prices.
      </h6>
    </div>
  </div>
  <div
    class="row"
    data-test="row"
  >
    <div
      class="col-md-6 imageBKGR"
      data-test="col"
    >
      <p
        id="services"
      />
      <h3>
        Chemicals
      </h3>
      <ul>
        <li>
          <h6>
            We use the highest professional grade chemicals available each week at every pool.
          </h6>
        </li>
        <li>
          <h6>
            You've heard the phrase "one size fits all" most likely, but unfortunately that's not always the case. That's especial true when it comes to keeping water balanced and the amount of chemical and time to make that happen. Your pool is a large investment, would you want a handyman to take care it that kind of knows what they're doing? Or do you want a professional that will be honest with how much it's going to cost, and take the time to care for that body of water, it's equipment, and keep your family safe?.
          </h6>
        </li>
      </ul>
    </div>
    <div
      class="col-md-6 text-center formatPic"
      data-test="col"
    >
      <div
        data-aos="zoom-in-right"
        data-aos-duration="1600"
        data-aos-easing="ease-in-sine"
        data-aos-offset="200"
        fluid="true"
      >
        <img
          alt="Wading Pool"
          fluid="true"
          height="100%"
          padding-top="10"
          src="WadingPool.jpg"
          width="100%"
        />
      </div>
    </div>
  </div>
  <div
    class="row"
    data-test="row"
  >
    <div
      class="col-md-6 text-center formatPic"
      data-test="col"
    >
      <div
        data-aos="zoom-in-left"
        data-aos-duration="1600"
        data-aos-easing="ease-in-sine"
        data-aos-offset="200"
      >
        <img
          alt="Wading Pool"
          fluid="true"
          height="100%"
          padding-top="10"
          src="1.jpg"
          width="100%"
        />
      </div>
    </div>
    <div
      class="col-md-6 imageBKGR"
      data-test="col"
    >
      <ul>
        <h3>
          Weekly service includes
        </h3>
        <li>
          <h5>
            Brushing walls and steps
          </h5>
        </li>
        <li>
          <h5>
            Empty all baskets
          </h5>
        </li>
        <li>
          <h5>
            Balance chemical
          </h5>
        </li>
        <li>
          <h5>
            Skim debris from pool floor and surface
          </h5>
        </li>
        <li>
          <h5>
            Check equipment
          </h5>
        </li>
      </ul>
    </div>
  </div>
  <footer
    class="page-footer footer"
    data-test="footer"
  >
    <div
      class="row"
      data-test="row"
    >
      <div
        class="col-lg-12 footer"
        data-test="col"
      >
        <br />
        <br />
        <br />
        <h5
          class="footer"
        >
           @copyright 2020. All rights Reserved
        </h5>
      </div>
    </div>
  </footer>
</DocumentFragment>
`;
